codespell_lib/data/dictionary.txt

```diff
@@ -19856,6 +19900,7 @@ linarly->linearly
 lincese->license
 lincesed->licensed
 linceses->licenses
+lineal->linear
```

Comment:
Checks failed because `lineal` is a valid word. I suggest you remove this.

* https://www.google.com/search?q=define+lineal
* https://www.lexico.com/definition/lineal
* https://www.ldoceonline.com/dictionary/lineal
* https://www.merriam-webster.com/dictionary/lineal